src/osd/PrimaryLogPG.cc

```diff
@@ -4288,8 +4288,11 @@ void PrimaryLogPG::execute_ctx(OpContext *ctx)
     }
     reply->add_flags(CEPH_OSD_FLAG_ACK | CEPH_OSD_FLAG_ONDISK);
     // append to pg log for dup detection - don't save buffers for now
```

Comment:
Is the "don't save buffers for now" bit still valid?

Reply:
Yes, it's still the same (by _the buffers_ I understand e.g. a write op's payload).

Reply:
I interpreted it as write op's _reply_ payload (i.e. out data).  If you believe the comment to be valid, I withdraw ;)